grpc-stubs/aio.pyi

```diff
@@ -0,0 +1,258 @@
+import typing
+from concurrent import futures
+from types import TracebackType
+from . import (
+    _Options,
+    Compression,
+    GenericRpcHandler,
+    ServerCredentials,
+    StatusCode,
+    ChannelCredentials,
+    CallCredentials,
+)
+
+"""Create Client"""
+
+# FIXME
+ClientInterceptor = typing.Any
+
+def insecure_channel(
+    target: str,
+    options: typing.Optional[_Options] = None,
+    compression: typing.Optional[Compression] = None,
+    interceptors: typing.Optional[typing.Sequence[ClientInterceptor]] = None,
+) -> Channel: ...
+def secure_channel(
+    target: str,
+    credentials: ChannelCredentials,
+    options: typing.Optional[_Options] = None,
+    compression: typing.Optional[Compression] = None,
+    interceptors: typing.Optional[typing.Sequence[ClientInterceptor]] = None,
+) -> Channel: ...
+
+"""Create Server"""
+
+def server(
+    migration_thread_pool: typing.Optional[futures.Executor] = None,
+    handlers: typing.Optional[typing.Sequence[GenericRpcHandler]] = None,
+    interceptors: typing.Optional[typing.Sequence[typing.Any]] = None,
```

Comment:
Could you please fix up this `typing.Any` here? The non-async version has a type for this.

Reply:
For aio, server interceptors are an experimental API, and this is actually how the docs specify the type: https://grpc.github.io/grpc/python/grpc_asyncio.html#create-server

Reply:
Ah cool, thanks for clarifying! Could you please add that in a comment to `interceptors`?